bigbluebutton-html5/private/config/settings.yml

```diff
@@ -282,6 +282,7 @@ public:
     enabled: false
     syncInterval: 60000
   allowOutsideCommands:
+    clearPublicChat: true
```

Comment:
Let's keep this as a false by default for now please, just like the other outside commands

Reply:
Oops, I meant to include it as false, will fix